### PR TITLE
Smoke tests: Large python notebook

### PR DESCRIPTION
**Pull request by testlabauto**
This test uses this [large python notebook](https://github.com/posit-dev/qa-example-content/blob/main/workspaces/large_py_notebook/spotify.ipynb)

It opens the notebook and runs it.  Then it scrolls though about 2/3 of the output collecting image refs to loosely ensure that the document generated correctly.

It is a pretty heavy test.  Not planning to run it on web and windows.  Tested on web and just running the notebook takes over a minute.

### QA Notes

All smoke tests should pass.

### CODE REVIEW

File: test/smoke/src/areas/positron/notebook/largePythonNotebook.test.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+		let app: Application;
+		let notebooks: PositronNotebooks;
+
+		before(async function () {
+			app = this.app as Application;
+			notebooks = app.workbench.positronNotebooks;
+			await PositronPythonFixtures.SetupFixtures(app);
+		});
+
+		it('Python - Large notebook execution [C...]', async function () {
```
Comment (midleman) [inline test/smoke/src/areas/positron/notebook/largePythonNotebook.test.ts:25]: did you want to add the TR case id?

Comment (testlabauto) [reply test/smoke/src/areas/positron/notebook/largePythonNotebook.test.ts:25]: oh yeah, thanks

Comment (testlabauto) [reply test/smoke/src/areas/positron/notebook/largePythonNotebook.test.ts]: done

**Review by jonvanausdeln — DISMISSED** [review]
LGTM!  It worked fine on my windows machine.

I DO think we should run it on windows and web.  Those will be our 2 biggest platforms for users.  I am sure however that to run on windows CI you will need to make changes to hide sidebars, etc.  This should be part of the nightly tests.

File: .github/workflows/positron-merge-to-main.yml [hunk 2]
```diff
@@ -80,7 +80,7 @@ jobs:
           POSITRON_PY_VER_SEL: 3.10.12
           POSITRON_R_VER_SEL: 4.4.0
         id: electron-smoke-tests
-        run: DISPLAY=:10 yarn ${{ env.SMOKETEST_TARGET }} --tracing --parallel --jobs 2 --skip-cleanup
```
Comment (midleman) [inline .github/workflows/positron-merge-to-main.yml:83]: have there been problems with 2 workers? i haven't seen any

Comment (testlabauto) [reply .github/workflows/positron-merge-to-main.yml:83]: oh, i thought you mentioned yesterday that PR runs were more unreliable than full test runs?  maybe I was not following you.

**Comment by testlabauto** [comment]
> LGTM! It worked fine on my windows machine.
> 
> I DO think we should run it on windows and web. Those will be our 2 biggest platforms for users. I am sure however that to run on windows CI you will need to make changes to hide sidebars, etc. This should be part of the nightly tests.

Ok, I will try on web and win and see if it can work with very large timeouts.

**Review by midleman — DISMISSED** [review]

**Comment by jonvanausdeln** [comment]
> > LGTM! It worked fine on my windows machine.
> > I DO think we should run it on windows and web. Those will be our 2 biggest platforms for users. I am sure however that to run on windows CI you will need to make changes to hide sidebars, etc. This should be part of the nightly tests.
> 
> Ok, I will try on web and win and see if it can work with very large timeouts.

If it gets to be too much, I'm okay with it not running on windows & web, but I think we should try.  If you do end up not running on those.. it would be good to leave a comment in the file with a quick explanation.. so someone (probably me) forgets and spends too much time trying again :)

**Comment by testlabauto** [comment]
> > > LGTM! It worked fine on my windows machine.
> > > I DO think we should run it on windows and web. Those will be our 2 biggest platforms for users. I am sure however that to run on windows CI you will need to make changes to hide sidebars, etc. This should be part of the nightly tests.
> > 
> > 
> > Ok, I will try on web and win and see if it can work with very large timeouts.
> 
> If it gets to be too much, I'm okay with it not running on windows & web, but I think we should try. If you do end up not running on those.. it would be good to leave a comment in the file with a quick explanation.. so someone (probably me) forgets and spends too much time trying again :)

Yeah, it doesn't seem to pass on web and windows.  I added a comment to that effect.

Its a pretty demanding test.  I hope we can just use it to keep an eye on notebook performance as dev proceeds.

**Pull request merged**